client/app/bundles/course/assessment/question/scribing/containers/ScribingQuestionForm/index.jsx

```diff
@@ -0,0 +1,288 @@
+import React, { PropTypes } from 'react';
+import { injectIntl, intlShape } from 'react-intl';
+import { reduxForm, Field, Form } from 'redux-form';
+import TextField from 'lib/components/redux-form/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+
+import MaterialSummernote from 'lib/components/MaterialSummernote';
+import ChipInput from 'lib/components/ChipInput';
+
+import styles from './ScribingQuestionForm.scss';
+import translations from './ScribingQuestionForm.intl';
+
+import { fetchScribingQuestion, createScribingQuestion, updateScribingQuestion } from '../../actions/scribingQuestionActionCreators';
+import { formNames } from '../../constants';
+
+const propTypes = {
+  dispatch: PropTypes.func.isRequired,
```

Comment:
It's better to pass in the exact functions that are needed by this component rather than passing in one whole dispatch function. Reason being you can stub out the respective props if you want to do testing.